### PR TITLE
Ansible 9.2.0: Dependencies, changelog and porting guide

### PR DESCRIPTION
**Pull request by gotmax23**

### CODE REVIEW

**Comment by gotmax23** [comment]
I uploaded the build artifacts to https://a.gtmx.me/ansible-builds/.

File: 9/CHANGELOG-v9.rst [hunk 1]
```diff
@@ -209,6 +1056,12 @@ community.zabbix
 - zabbix_api_info module added
 - zabbix_user module - add current_passwd optional parameter to enable password updating of the currently logged in user (https://www.zabbix.com/documentation/6.4/en/manual/api/reference/user/update)
 
+dellemc.openmanage
+~~~~~~~~~~~~~~~~~~
+
+- Ansible lint issues are fixed for the collections.
+- Module ``redfish_storage_volume`` is enhanced to support reboot options and job tracking operation.
+
```
Comment (mariolenz) [inline 9/CHANGELOG-v9.rst:1064]: here

File: 9/CHANGELOG-v9.rst [hunk 2]
```diff
@@ -339,6 +1192,7 @@ cisco.iosxr
 cisco.ise
 ~~~~~~~~~
 
+- Added missing import re in endpoint module
```
Comment (mariolenz) [inline 9/CHANGELOG-v9.rst:1195]: here

File: 9/CHANGELOG-v9.rst [hunk 3]
```diff
@@ -4090,6 +4944,11 @@ netapp.ontap
 - netapp.ontap.na_ontap_active_directory_domain_controllers - NetApp ONTAP configure active directory preferred domain controllers
 - netapp.ontap.na_ontap_ems_config - NetApp ONTAP module to modify EMS configuration.
 
+netbox.netbox
+~~~~~~~~~~~~~
+
+- netbox.netbox.netbox_config_template - Creates, updates, or removed a config template from NetBox
+
```
Comment (mariolenz) [inline 9/CHANGELOG-v9.rst:4951]: and here

**Comment by gotmax23** [comment]
https://github.com/ansible-community/ansible-build-data/releases/tag/9.2.0

**Review by mariolenz — COMMENTED** [review]
Why are there new changelog entries for past releases? Is this a bug, or does it fix a bug?

Or am I completely wrong here?

File: 9/CHANGELOG-v9.rst (hunk 1 above)
Comment (felixfontein) [reply 9/CHANGELOG-v9.rst:1064]: This usually happens if collections have a new release in which they also modify changelog entries for older releases. For example, for dellemc.openmanage, which was updated from 8.5.0 to 8.7.0, look at the comparison: https://github.com/dell/dellemc-openmanage-ansible-modules/compare/v8.5.0..v8.7.0#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43c - you can see that they modified the 8.5.0 changelog as well *after* the 8.5.0 release. If you look at the diff for changelogs/changelog.yaml, you can see that they fixed a broken section name, and a typo in a corresponding fragment.

**Pull request merged**